common/src/test/java/apoc/util/UtilIT.java

```diff
@@ -63,10 +57,26 @@ public void redirectShouldWorkWhenProtocolNotChangesWithUrlLocation() throws IOE
         assertTrue(page.contains("<title>Google</title>"));
     }
 
-    @Ignore
+    @Test
+    public void redirectWithBlockedIPsWithUrlLocation() {
+        List<IPAddressString> blockedIPs = List.of(new IPAddressString("127.168.0.1/8"));
+
+        Config neo4jConfig = mock(Config.class);
+        when(neo4jConfig.get(GraphDatabaseInternalSettings.cypher_ip_blocklist)).thenReturn(blockedIPs);
+
+        setUpServer(neo4jConfig, "http://127.168.0.1");
+        String url = getServerUrl();
+
+        IOException e = Assert.assertThrows(IOException.class,
+                () -> Util.openInputStream(url, null, null, null)
+        );
+        TestCase.assertTrue(e.getMessage().contains("access to /127.168.0.1 is blocked via the configuration property internal.dbms.cypher_ip_blocklist"));
```

Comment:
Hm...what does the slash here signify?

Reply:
This is the InetAddress from Java, it isn't added by us, it is what the host URL got resolved as in a toString. It is because it would print out hostName/address, but in this particular case there is no hostname e.g www.google.com/64.34.119.12 is possible. I think this is useful for IPs that have hostnames as a user may understand the text part more. e.g www.evilattacker.com/64.34.119.12 is easier to read than 64.34.119.12